src/components/spreadsheet/utils/config-tables.js

```diff
@@ -3038,6 +3208,30 @@ export const TABLES_DEFINITIONS = {
                 cellRenderer: BooleanCellRenderer,
                 getQuickFilterText: excludeFromGlobalFilter,
             },
+            {
+                id: 'Properties',
+                field: 'properties',
+                editable: isEditable,
+                cellStyle: editableCellStyle,
+                valueGetter: propertiesGetter,
+                cellRenderer: PropertiesCellRenderer,
+                minWidth: 300,
+                getQuickFilterText: excludeFromGlobalFilter,
+                valueSetter: (params) => {
+                    params.data.properties = params.newValue;
+                    return params;
+                },
+                cellEditor: SitePropertiesEditor,
```

Comment:
can you please rename the cellEditor to `PropertiesEditor` since now it is used for all equipments